src/Iot.Device.Bindings.SkiaSharpAdapter/Directory.Build.props

```diff
@@ -0,0 +1,13 @@
+<Project>
```

Comment:
overall do we need to update samples to mention that package should be used?

Reply:
It's done (or will be done) for all projects requiring the imaging library.